src/microsoft/office/license/edit/password/microsoft-office-edit-password.html

```diff
@@ -7,7 +7,9 @@
         <div data-wizard-step
              data-wizard-step-valid="formEditPassword.$valid && (!editPwdCtrl.model.password || editPwdCtrl.model.password === editPwdCtrl.password.confirm) && !editPwdCtrl.loaders.edit">
 
-            <div class="loader text-center" data-ng-if="editPwdCtrl.loaders.infos || editPwdCtrl.loaders.edit"></div>
+            <div class="text-center" data-ng-if="editPwdCtrl.loaders.infos || editPwdCtrl.loaders.edit">
+                <oui-loader inline="true"></oui-loader>
```

Comment:
In general, i see a lot a loader with `inline` in a `text-center` container. 
I wonder why the base loader (with `display: block`) is not centered ?

In this case, we will just get:
`<oui-loader data-ng-if="editPwdCtrl.loaders.infos || editPwdCtrl.loaders.edit"></oui-loader>`

Reply:
I think it's a matter of perspective. 

We have to keep in mind that oui-loader will be use everywhere, so in my opinion it is positif to keep this flexibility.
I'm not sure to set it as a block will fit in everycase such as a loader in a table ?
But may be it would be interesting to discuss of it together!

Reply:
By default, it has display block (http://master.ui-kit.ovh/#!/ovh-ui-kit/loader). The problem is that it's align left in a block container. 
I said that because 90% of the use of `<oui-loader>` in manager are inline loaders in a text-centered container for a page/view loading.

For the use we make of it, `.oui-loader__container` will be more useful with a `margin: auto`. And it will not impact the inline one :)

Out of curiosity, do you have an example of its use in a table ?

Reply:
Yes, all the good ideas are welcome in the UI-Kit! Feel free to contribute youself with an awesome PR :) We will see what other contributers think about it.

In the tables, I think that the loaders must be medium centered.

Reply:
I think you agree most of the time we use centered loader. Extremely rare are the use case of none centered. I only have in mind few example in cloud : 

- here are one : https://github.com/ovh-ux/ovh-manager-cloud/blob/master/client/app/cloud/project/compute/infrastructure/cloud-project-compute-infrastructure.html#L1039-L1041
 - and in that aliens too : https://github.com/ovh-ux/ovh-manager-cloud/blob/master/client/app/cloud/project/compute/infrastructure/virtualMachine/addEdit/cloud-project-compute-infrastructure-virtualMachine-addEdit.html

Reply:
@neolitec Ok, i'll submit a PR for this :)

@whispyy In your two cases, the loaders have a `d-inline-block` class. It takes the parent alignment, so it's good.

Anyway, instead of the `d-inline-block` class, should it not be `<oui-loader inline="true" size="s"></oui-loader>` ?

Reply:
Why not just add a parameter on the component that let us decide what position we'd like to have?

Left, right, center, etc.

It is not uncommon to see this kind of parameter and it is very clear and simple to use.